.github/workflows/pypi.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       run: |
         python -m build --sdist --wheel
     - name: Publish servicex to PyPI
-      uses: pypa/gh-action-pypi-publish@v1.3.1
+      uses: pypa/gh-action-pypi-publish@v1.6.4
       with:
-        user: __token__
-        password: ${{ secrets.pypi_password_servicex }}
+        password: ${{ secrets.pypi_password }}
```

Comment:
You don't need to specify the `user: __token__` here because

https://github.com/pypa/gh-action-pypi-publish/blob/7eb3b701d11256e583f5b49899c5e7203deab573/README.md?plain=1#L183-L193

> ### Specifying a different username
>
> The default username value is `__token__`. If you publish to a custom
> registry that does not provide API tokens, like `devpi`, you may need to
> specify a custom username and password pair. This is how it's done.
> 
> ```yml
>   with:
>     user: guido
>     password: ${{ secrets.DEVPI_PASSWORD }}
> ```